rucio clis: undefined name 'result' in def info_rule

Motivation
----------
```
$ flake8 lib/rucio/client/clis/rucio.py
lib/rucio/client/clis/rucio.py:2225:16: F821 undefined name 'result'
```
read  `bin/rucio` instead of `lib/rucio/client/clis/rucio.py`

```
        if args.estimate_ttc:
            if result[0] > 0:
                print("Expected transfer start in  %0.2f minutes." % (rule['estimated_start_in'] / 60.))
                print("Expected transfer end in    %0.2f minutes." % (rule['estimated_end_in'] / 60.))
            else:
                print('Couldn\'t calculate the TTC for any of the transfers. No sources selected yet?')
```

but `result` doesn't exist.

--- a/bin/rucio
+++ b/bin/rucio
@@ -1314,7 +1314,7 @@ def info_rule(args):
         print("End of life:                %s" % rule['eol_at'])
         print("Child Rule Id:              %s" % rule['child_rule_id'])
         if args.estimate_ttc:
-            if result[0] > 0:
+            if 'estimated_start_in' in rule and 'estimated_end_in' in rule:
                 print("Expected transfer start in  %0.2f minutes." % (rule['estimated_start_in'] / 60.))
                 print("Expected transfer end in    %0.2f minutes." % (rule['estimated_end_in'] / 60.))
             else:

